apps/api/v2/src/modules/organizations/controllers/inputs/validator/isEmailStringOrArray.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from "@nestjs/common";
+import { ValidationArguments, ValidatorConstraint, ValidatorConstraintInterface } from "class-validator";
+
+@Injectable()
+@ValidatorConstraint({ name: "isSingleEmailOrEmailArray" })
+export class IsSingleEmailOrEmailArray implements ValidatorConstraintInterface {
```

Comment:
Copy functionality of the V1 users endpoint. Add validation to ensure that emails are being passed in the body instead of using Zod.